CommandLineTool.yml

```diff
@@ -1099,9 +1099,10 @@ $graph:
   name: TimeLimit
   extends: ProcessRequirement
   doc: |
-    Set an upper limit on the execution time of a CommandLineTool or
-    ExpressionTool.  A tool execution which exceeds the time limit may
-    be preemptively terminated and considered failed.  May also be
+    Set an upper limit on the execution time of a process
+    (e.g. a CommandLineTool or a Workflow).  A process whose
```

Comment:
The fact that this is in `CommandLineTool.yml` and not `Process.yml` implies that it is for `CommandLineTool`s only. Therefore we should move it to `Process.yml`

Reply:
@mr-c I found no changelog for Process.yml. I'll add a section that appears above concepts

Reply:
There are two CWL standards, so each gets a changelog:
1. CWL command line tools
2. CWL workflows (made from CWL CLTs and ExpressionTools)

So changes to `Process.yml` need to represented in both changelogs

Reply:
#44